app.css

```diff
@@ -0,0 +1,56 @@
+* {
+  direction: rtl;
+  padding:0px;
+  margin:0px;
+}
```

Comment:
1. + 2. I think for now each increment should be a section in the page. Lets say each of the following should be an increment: structure, nabber, slider + button, the 3 images + text, testimonials, form.
3. I mean that visually the logo should be aligned to the right of the 960px div, while the others should be aligned to the left of the div. How to implement it is up to you.
4. Let's do that for now. Probably will change it along the road.